plugins/wake/wake.plugin.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+# oh-my-bash.module: wake-on-lan wrapper + autocompletion
+# wake.plugin.sh
+# Author: hoek from 0ut3r.space
+# Based on oh-my-zsh wake plugin
```

Comment:
Could you include the corresponding commit id of the upstream oh-my-zsh? It can be used to import the upstream change when the upstream plugin is changed.